inference/python/infbench/cutlass.py

```diff
@@ -93,33 +191,38 @@ class cutlassSgemmLoader(dataset.loader):
     checkAvailable = True
 
     def __init__(self, dataDir):
-        self.M = 10000
-        self.N = 8000
+        self.M = 1000
+        self.N = 800
         self.K = 10000
+        nPoints = 20
+        self.A = [0 for i in range(nPoints)]
+        self.B = [0 for i in range(nPoints)]
 
     @property
     def ndata(self):
-        return 1000
+        return 20
```

Comment:
May as well go all the way down to 1. I don't think the experiment gains anything by having more inputs and it affects our start times and memory usage. If we try bigger matrices for whatever reason, even 20 is a problem